### PR TITLE
Mech balistic gun "VJP"

### PR DESCRIPTION
**Pull request by TheShown911**
## About The Pull Request

Comming to you at a modest 100 plasteel + tax, the modified PK fited for mech use, "VJP" fires in 15 round bursts and regenerates ammo boxes at a nice 500 energy per box. Due to rushed production it has *some* accuracy problems.
robit1
robit2

## Why It's Good For The Game

Mechs truly lack a balistic gun and very destructive weaponry in general.

## Changelog
:cl:
add: Added SA "VJP" - a modified PK fited for mechs.
/:cl:

### CODE REVIEW

File: code/modules/mechs/equipment/combat.dm [hunk 1]
```diff
@@ -76,3 +76,53 @@
...
+	return "[get_ammo()]/[ammo_magazine.max_ammo]"
+
+/obj/item/weapon/gun/projectile/get_hardpoint_status_value()
+	if(ammo_magazine)
+		return get_ammo()/ammo_magazine.max_ammo
+	return null
+
+/obj/item/mech_equipment/mounted_system/balistic_gun/pk
+	name = "SA \"VJP\""
+	desc = "A reverse engineered Pulemyot Kalashnikova fitted for mech use. Fires in 15 round bursts. Horribly inacurate, but packs quite a punch."
```
Comment (Firefox13) [inline code/modules/mechs/equipment/combat.dm:90]: inaccurate

File: code/modules/mechs/equipment/combat.dm [hunk 2]
```diff
@@ -76,3 +76,53 @@
...
+	holding_type = /obj/item/weapon/gun/projectile/automatic/lmg/pk/mounted/mech
+	restricted_hardpoints = list(HARDPOINT_LEFT_HAND, HARDPOINT_RIGHT_HAND)
+	restricted_software = list(MECH_SOFTWARE_WEAPONS)
+	origin_tech = list(TECH_COMBAT = 5, TECH_MAGNET = 3)
+	matter = list(MATERIAL_PLASTEEL = 60)
+	spawn_blacklisted = TRUE
+
+/obj/item/weapon/gun/projectile/automatic/lmg/pk/mounted/mech
+	name = 	"SA \"VJP\""
+	desc = "A reverse engineered Pulemyot Kalashnikova fitted for mech use. Fires in 15 round bursts. Horribly inacurate, but packs quite a punch."
```
Comment (Firefox13) [inline code/modules/mechs/equipment/combat.dm:101]: inaccurate

File: code/modules/mechs/equipment/combat.dm (hunk 1 above)
Comment (TheShown911) [inline code/modules/mechs/equipment/combat.dm:90]:
```suggestion
	desc = "A reverse engineered Pulemyot Kalashnikova fitted for mech use. Fires in 15 round bursts. Horribly inaccurate, but packs quite a punch."
```

File: code/modules/mechs/equipment/combat.dm (hunk 2 above)
Comment (TheShown911) [inline code/modules/mechs/equipment/combat.dm:101]:
```suggestion
	desc = "A reverse engineered Pulemyot Kalashnikova fitted for mech use. Fires in 15 round bursts. Horribly inaccurate, but packs quite a punch."
```

Comment (TheShown911) [reply code/modules/mechs/equipment/combat.dm]: Fixed.

File: code/modules/mechs/equipment/combat.dm (hunk 1 above)
Comment (TheShown911) [reply code/modules/mechs/equipment/combat.dm]: Fixed.

**Review by TorinoFermic — APPROVED** [review]

**Review by Hopekz — APPROVED** [review]
We'll see how it goes balancing-wise once we've seen this being used.
I smell a future nerf

**Pull request merged**